data/web/lang/lang.en.php

```diff
@@ -821,6 +821,7 @@
 $lang['quarantine']['high_danger'] = "High";
 $lang['quarantine']['danger'] = "Danger";
 $lang['quarantine']['spam_score'] = "Score";
+$lang['quarantine']['notified'] = "notified";
```

Comment:
Hello, I would write "Notified" instead of "notified" based on the case of other words.  
Additionally, this is something you may want to change on your DE translation as well!